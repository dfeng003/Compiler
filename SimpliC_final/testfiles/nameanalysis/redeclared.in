int gcd1(int a, int b) {
int a = 4;
while (a != c) {
a = a - b;
}
print();
return d;
}

int print(){
}

int gcd1(){
}