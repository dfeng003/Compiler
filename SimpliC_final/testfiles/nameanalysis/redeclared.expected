Error at line 2: symbol 'a' is already declared!
Error at line 3: symbol 'c' is not declared
Error at line 6: the calling of symbol 'print' is wrong
Error at line 7: symbol 'd' is not declared
Error at line 10: symbol 'print' is already declared!
Error at line 13: symbol 'gcd1' is already declared!
